framework/axSGreaseSkeleton.js

```diff
@@ -11,6 +11,59 @@
 
 /*** Functions for common tweaks. ***/
 
+/**
+ * Adds text to the given live region, and clears it a second later so its no
+ * longer in the virtual buffer.
+ * @param {string} regionid an id of a region.
+ */
+function announce(text, regionId) {
+	getLiveRegion(regionId)
+		.then((region) => {
+			region.innerText = text;
+			setTimeout(() => {
+				region.innerText = '';
+			}, 1000);
```

Comment:
I really wish we could convert this into an async function with awaits, but there's still no native timeout Promise, so we'd just be trading one ugliness for another. :(